src/Components/Nav/Nav.jsx

```diff
@@ -3,14 +3,14 @@ import "./Nav.css";
 function Nav() {
   return (
     <nav className="navigation">
```

Comment:
Please add white space between lines 4 & 5

Reply:
again VsCode thinks it's better this way, and I don't see why either, it already has an indentation